tox.ini

```diff
@@ -1,56 +1,48 @@
 [tox]
-envlist = {py35,py36,py38,py27}{,-win},lint
-
+requires = tox>=4
+toxworkdir=/tmp/.tensile-tox
+labels =
+    precommit = lint, unittest
+
+[vars]
+client_path = {envdir}/client/0_Build/client/tensile_client
+options =
+    --verbose \
+    --basetemp={envtmpdir} \
+    --color=yes \
+    --numprocesses=8 \
+    --prebuilt-client={[vars]client_path}
 
 [testenv]
-platform = linux
-# Some versions of Pytest versions have a bug:
-# https://github.com/pytest-dev/pytest/issues/5971 which causes the whole test
-# process to crash if a multiprocessing job has an exception. Fixed in 5.3.3.
+basepython = python3
 deps =
     -r{toxinidir}/requirements.txt
     joblib
-    pytest>=5.4.1
-    pytest-xdist>=1.32.0
     filelock
+    pytest>=7.2.0
+    pytest-xdist>=3.3.0
```

Comment:
Looks like some requirements got updated versions, probably a good idea to update the dependancy list on the wiki as well